lib/meson.build

```diff
@@ -70,7 +70,6 @@ xnvmelib_source = [
   'xnvme_kvs.c',
   'xnvme_lba.c',
   'xnvme_libconf.c',
-  'xnvme_libconf_entries.c',
```

Comment:
I don't understand why this is being removed, don't we need it?

Reply:
It is imported by ``xnvme_libconf.c``.
Previously then the file provided the implementation backing the ``extern struct``. However, now that this is removed and instead retrieved via a getter, then ``xnvme_libconf_entries.c`` is not translated, instead ``xnvme_libconf.c`` is translated which includes ``xnvme_libconf_entries.c``.

Reply:
Great